src/example/p4p5/profiles/uninstall-plone4/cssregistry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
It is not GenericSetup that takes care of this. It is the portal_quickinstaller who, if all goes well, keeps track of *some* things that are installed, and removes them when you uninstall the packages. The 'some' part is one of the reasons I want to remove the quick installer (the biggest other reason is that having two ways for installing is, and two places for keeping this information, is not making our add-on story clearer.) It is better to have an uninstall profile that explicitly lists what should be removed.
If someone wants to argue otherwise, please add your thoughts to the plip at https://github.com/plone/Products.CMFPlone/issues/1340